web/modeltr/connection.py

```diff
@@ -35,7 +37,8 @@ def __init__(self, **kwargs):
         for i in range(Settings.app['retries']['db_connection']):
             try:
                 self.async_mode = True if 'async_mode' in kwargs else False
-                self.client = psycopg2.connect(kwargs['dsn'], async_=self.async_mode)
+                temp_async_mode = 1 if self.async_mode else 0
```

Comment:
Please have a look here:
https://www.geeksforgeeks.org/python-ways-to-convert-boolean-values-to-integer/